<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>findshop_admin</class>
 <widget class="QDialog" name="findshop_admin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>480</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton {
    background-color: #F5F2E7; /* –Ø—Ä–∫–æ-–∫—Ä–∞—Å–Ω—ã–π */
    color: black;              /* –ß–µ—Ä–Ω—ã–π —Ç–µ–∫—Å—Ç */
 
}

QPushButton:hover {
    background-color: #01eb00; /* –¢–µ–º–Ω–µ–µ –ø—Ä–∏ –Ω–∞–≤–µ–¥–µ–Ω–∏–∏ */
}

QPushButton:pressed {
    background-color: #01a800 /* –ï—â—ë —Ç–µ–º–Ω–µ–µ –ø—Ä–∏ –Ω–∞–∂–∞—Ç–∏–∏ */
}

</string>
   </property>
   <property name="text">
    <string>–ì–æ—Ç–æ–≤–æ</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>100</y>
     <width>641</width>
     <height>361</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QWidget" name="layoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>621</width>
      <height>311</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;–ù–∞–∑–≤–∞–Ω–∏–µ –º–∞–≥–∞–∑–∏–Ω–∞&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="name_3">
         <property name="placeholderText">
          <string>–í–≤–µ–¥–∏—Ç–µ –Ω–∞–∑–≤–∞–Ω–∏–µ –º–∞–≥–∞–∑–∏–Ω–∞</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="labe">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;–ê–¥—Ä–µ—Å&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="adress_3">
         <property name="placeholderText">
          <string>–í–≤–µ–¥–∏—Ç–µ –∞–¥—Ä–µ—Å –º–∞–≥–∞–∑–∏–Ω–∞</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;–†–∞–±–æ—á–∏–µ —á–∞—Å—ã&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="workingtime_3">
         <property name="placeholderText">
          <string>–í–≤–µ–¥–∏—Ç–µ —Ä–∞–±–æ—á–∏–µ —á–∞—Å—ã –º–∞–≥–∞–∑–∏–Ω–∞</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>621</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:36pt;&quot;&gt;üõí –ü–æ–∏—Å–∫ –º–∞–≥–∞–∑–∏–Ω–∞ üõí&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>550</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">

QPushButton { background-color: #F5F2E7; /* –Ø—Ä–∫–æ-–∫—Ä–∞—Å–Ω—ã–π */ color: black; /* –ß–µ—Ä–Ω—ã–π —Ç–µ–∫—Å—Ç */ } QPushButton:hover { background-color: #D32F2F; /* –¢–µ–º–Ω–µ–µ –ø—Ä–∏ –Ω–∞–≤–µ–¥–µ–Ω–∏–∏ */ } QPushButton:pressed { background-color: #B71C1C; /* –ï—â—ë —Ç–µ–º–Ω–µ–µ –ø—Ä–∏ –Ω–∞–∂–∞—Ç–∏–∏ */ }
</string>
   </property>
   <property name="text">
    <string>–í–µ—Ä–Ω—É—Ç—å—Å—è –Ω–∞–∑–∞–¥</string>
   </property>
  </widget>
  <zorder>groupBox</zorder>
  <zorder>label</zorder>
  <zorder>pushButton</zorder>
  <zorder>pushButton_2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
